Downloader.m

```diff
@@ -88,6 +88,10 @@ - (void)URLSession:(NSURLSession *)session downloadTask:(NSURLSessionDownloadTas
 
 - (void)URLSession:(NSURLSession *)session downloadTask:(NSURLSessionDownloadTask *)downloadTask didFinishDownloadingToURL:(NSURL *)location
 {
+    NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *)downloadTask.response;
+    if (!_statusCode) {
+        _statusCode = [NSNumber numberWithLong:httpResponse.statusCode];
```

Comment:
I've added the download status code fetching in my own PR, https://github.com/CommercialTribe/react-native-fs/pull/4
 maybe you could remove it from this PR to avoid conflicts